CRM/Report/Form.php

```diff
@@ -2073,6 +2077,27 @@ public function whereClause(&$field, $op, $value, $min, $max) {
     return $clause;
   }
 
+  /**
+   * Get SQL where clause for contact subtypes
+   * @param string $field
+   * @param mixed $value
+   * @param string $op SQL Operator
+   *
+   * @return string
+   */
+  public function whereSubtypeClause($field, $value, $op) {
+    $clause = '( ';
+    $subtypeFilters = count($value);
+    for ($i = 0; $i < $subtypeFilters; $i++) {
+      $clause .= "{$field['dbAlias']} LIKE '%$value[$i]%'";
+      if ($i !== ($subtypeFilters - 1)) {
+        $clause .= " OR ";
```

Comment:
@eileenmcnaughton will the OR here cause performance issues possibly?

Reply:
I'll give it a whirl - but my first instinct here is that we are defining the field metadata incorrectly in some way or this wouldn't be needed